Symfony/CS/Config/Config.php

```diff
@@ -22,6 +22,8 @@
  */
 class Config implements ConfigInterface
 {
+    const CACHE_FILE = '.php_cs.cache';
```

Comment:
now it is not a cache fie, but default cache cache. cache file is `->$cacheFile` ;)